<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FundWindow</class>
 <widget class="QMainWindow" name="FundWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>1316</width>
      <height>866</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>360</y>
       <width>1313</width>
       <height>1</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
background-color: red;
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="fund_l_name">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>370</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>持股机构：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fund_cb">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>370</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="report_cb">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>370</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="report_l_name">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>370</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>报告日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string>持股机构总览</string>
     </property>
    </widget>
    <widget class="QTableView" name="ihdetail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1291</width>
       <height>316</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="phdetail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>1291</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string>持仓机构明细</string>
     </property>
    </widget>
    <widget class="QLabel" name="create_date">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>采集时间：2025/10/24 15:35:33</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
